test/integration/services/data/test-update-reduction-status-by-ids.js

```diff
@@ -0,0 +1,46 @@
+const expect = require('chai').expect
+
+const appWorkloadOwnerHelper = require('../../../helpers/data/app-workload-owner-helper')
+const appReductionsHelper = require('../../../helpers/data/app-reductions-helper')
+
+const updateReductionStatusByIds = require('../../../../app/services/data/update-reduction-status-by-ids')
+const getAllOpenReductions = require('../../../../app/services/data/get-all-open-reductions')
+
+var inserts = []
+
+describe('services/data/get-all-open-reductions', function () {
```

Comment:
Copy and paste? Also in the test it() below

Reply:
How embarassing, fixed